apps/web/project.json

```diff
@@ -77,6 +77,30 @@
         "command": "graphql-codegen --config apps/web/codegen.yml"
       }
     },
+    "dev/init": {
+      "executor": "@nrwl/workspace:run-commands",
+      "options": {
+        "commands": [
+          "yarn get-secrets api web",
```

Comment:
Can't these be `echo` instead of invoking node?

Reply:
I just assumed `echo` was not portable to windows. Fixing...